modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -70,7 +70,7 @@
 	height: 100%;
 }
 
-.sidebar {
+.content-dashboard.sidebar {
```

Comment:
Since you're adding the `content-dashboard` class I probably wrap children elements inside of it

```
.content-dashboard.sidebar {
   .sidebar-body {
      padding-top: 0

      & > .sidebar-section {
         margin-bottom: 16px;
      }
   }
}
```

Reply:
Sure!

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/314/commits/14d4c556cbdf60681ff3cd61ec763446012bf510